documentation/modules/migrating-virtual-machines-cli.adoc

```diff
@@ -91,7 +95,7 @@ spec:
     namespace: {namespace}
 EOF
 ----
-<1> Specify the vSphere API end point, for example, `https://<vcenter.host.com>/sdk`.
+<1> Specify the vSphere API end point, for example, `\https://<vcenter.host.com>/sdk`.
 <2> Specify the name of the VMware `Secret` CR.
```

Comment:
'\' before 'https' - should it be there?

Reply:
Yes. Example URLs need to be escaped so that they do not turn into clickable links.

Reply:
Just checked. It looks like the escape character is not working, so I will take it off. I will have to add it when we start publishing downstream docs.